FOR REVIEW: Adding Endgame stuff (#335)
makerdao/governance-manual: delegation/avc.md

# Aligned Voter Committees (AVC)

{% hint style="warning" %} This documentation describes ongoing changes in functionality and processes. Be aware that parts may be inaccurate or out of date. {% endhint %}

> Aligned Voter Committees (AVCs) are organized groups of MKR holders who have a shared strategic vision for the Maker protocol. Its members must be [Alignment Conservers](alignment-conservers.md).

Reviewer: ```suggestion
Aligned Voter Committees (AVCs) are organized groups of MKR holders who have a shared strategic vision for the Maker protocol whose members are [Alignment Conservers](alignment-conservers.md).
```

AVCs have two main tasks in governance:
1. To propose modifications to the [Scope Artifacts](../governance/scopes-and-artifacts.md) to strengthen them over time.

> 2. To ensure that all [Aligned Delegates (ADs)](aligned-delegates.md) that are affiliated with them, execute the AVC's strategic vision when voting on governance issues.   

Reviewer: ```suggestion
2. To ensure that all [Aligned Delegates (ADs)](aligned-delegates.md) that are affiliated with them execute the AVC's strategic vision when voting on governance issues.   
```

## Overview

Each AVC produces an Aligned Governance Strategy document and five Aligned Scope Artifact documents with proposed updates to the Scope Artifact of each [Scope](../governance/list-of-scopes.md). The Aligned Governance Strategy document provides general guidelines for how Aligned Delegates that follow their governance strategy should vote.

> The documents produced by the AVCs abstract away the minutiae of day-to-day governance. Since Aligned Delegates execute AVC strategies, they allow MKR Holders to be reasonably informed and secure with regard to the general direction of the chosen delegate.

Reviewer: Not really a non-sequitur, but looks like one at a glance

According to their own strategy, AVCs fall somewhere in a spectrum that goes from **Hawk** to **Dove**:
  - Hawkish AVCs focus more on short-term income and asset accumulation, tending to prefer higher spreads and lower budgets
  - Dovish AVCs focus more on long-term growth and innovation, tending to prefer lower spreads and higher budgets
  - Balanced AVCs try to find a middle ground between the two

> The Hawk/Dove is a continuous spectrum, so many different hawkish or dovish AVCs can exist that are more or less balanced.

Reviewer: ```suggestion
The Hawk/Dove spectrum is a continuous, so many different hawkish or dovish AVCs can exist that are more or less balanced.
```

## Eligibility Requirements

In order for a voter committee to be considered an AVC, its members must be Alignment Conservers (AC) and verified MKR holders.

To become Active AVCs, AVCs are subject to additional eligibility requirements. For each quarterly governance cycle they must:
   - Create one Aligned Governance Strategy document and five Aligned Scope Proposals
   - Hold two AVC Subcommittee meetings for each scope

The [Governance Scope](https://mips.makerdao.com/mips/details/MIP113) recognizes Active AVCs. It can revoke this recognition anytime the AVC fails to meet all eligibility requirements.

## Member Participation Rewards

Member Participation Rewards is a compensation scheme available to AVC members. It considers their participation in each scope.
To be eligible for compensation, AVC members must:
 - Attend the two AVC Subcommittee Meetings
 - Co-author the Aligned Governance Strategy

> AVC Members who fulfill these eligibility requirements and have the highest verified MKR ownership are eligible for compensation.

Reviewer: So only one AVC member is eligible for compensation, the one with the highest verified MKR ownership?

## Visibility

> The [Easy Governance Frontend (EGF)](../governance/easy-governance-frontend.md) displays AVCs on a list that ranks them by their voter weight. The voter weight is calculated by adding the verified MKR holdings of members plus the MKR delegated to delegates that follow the AVC's Governance Strategy.

Reviewer: ```suggestion
The [Easy Governance Frontends (EGFs)](../governance/easy-governance-frontend.md) display AVCs on a list that ranks them by their voter weight. The voter weight is calculated by adding the verified MKR holdings of members plus the MKR delegated to delegates that follow the AVC's Governance Strategy.
```
